src/css/profile/wearable/changeable/theme-circle/layout.less

```diff
@@ -133,6 +133,7 @@ body {
 		border: 0 none;
 		padding: 0;
 		text-indent: -9999px;
+		z-index: 10;
```

Comment:
did you consider setting z-index to -1 for ui-listview
to lower priority of it to have more universal solution?

Reply:
Is it a more universal solution to lower the priority of the ui-listview?
If it is correct, I will change it.

Reply:
yes, I was thinking about this as a more universal solution @TomaszLukawskiSam how do you think?

Reply:
@lslachciak 
I think, @HunseopJeong 's solution is more stable. The listview is a key widget and this modification can have unexpected issues.